src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -180,6 +217,16 @@ def description_wrapper(description)
     end
   end
 
+  def webui2_description_wrapper(description)
```

Comment:
Why is this a 10 lines helper? It's always a `p.description-text` and the content is either `simple_format(description)` or `<i>No description set</i>`

Reply:
@hennevogel, you want to kill this helper? if not, this helper could be like:

```ruby
def webui2_description_wrapper(description)
  content_tag(:p, id: 'description-text', class: description.blank? ? '' : 'plain') do
    description.blank? ? content_tag(:i, 'No description set') : simple_format(description)
  end
end
```

less lines :smile_cat: